### PR TITLE
Create test-snap for cronjob

### PR DESCRIPTION
**Pull request by bowensanders**

### CODE REVIEW

File: packages/site/src/features/cronjob-snap/Cronjob.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import { useInvokeMutation } from '../../api';
+import { getSnapId } from '../../utils/id';
+
+const CRONJOB_SNAP_ID = 'npm:@metamask/test-snap-cronjob';
+const CRONJOB_SNAP_PORT = 8010;
+
+export const Cronjob: FunctionComponent = () => {
+  const [invokeSnap, { isLoading, data, error }] = useInvokeMutation();
+
+  const handleFireCronjob = () => {
```
Comment (FrederikBolding) [inline packages/site/src/features/cronjob-snap/Cronjob.tsx:13]: I don't think we need these functions and buttons to trigger the cronjob dialog?

File: packages/cronjob/snap.manifest.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      "jobs": [
+        {
+          "expression": "* * * * *",
+          "request": {
+            "method": "snap_dialog"
+          }
+        }
+      ]
+    },
+    "endowment:rpc": {
```
Comment (FrederikBolding) [inline packages/cronjob/snap.manifest.json:31]: It doesn't need `endowment:rpc`

File: packages/cronjob/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```
[the remark this answers is not in the document]
Comment (bowensanders) [reply packages/cronjob/CHANGELOG.md:7]: Fixed :)

File: packages/cronjob/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { OnCronjobHandler, OnRpcRequestHandler } from '@metamask/snaps-types';
+import { panel, text, heading } from '@metamask/snaps-ui';
+
+export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
```
[the remark this answers is not in the document]
Comment (bowensanders) [reply packages/cronjob/src/index.ts]: fixed

File: packages/cronjob/snap.manifest.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      }
+    }
+  },
+  "initialPermissions": {
+    "endowment:cronjob": {
+      "jobs": [
+        {
+          "expression": "* * * * *",
+          "request": {
+            "method": "snap_dialog"
```
[the remark this answers is not in the document]
Comment (bowensanders) [reply packages/cronjob/snap.manifest.json]: fixed

File: packages/site/src/features/cronjob-snap/Cronjob.tsx (hunk 1 above)
Comment (bowensanders) [reply packages/site/src/features/cronjob-snap/Cronjob.tsx:13]: fixed

File: packages/cronjob/snap.manifest.json (hunk 2 above)
Comment (bowensanders) [reply packages/cronjob/snap.manifest.json]: fixed

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**